### DISCUSSION TITLE
budibase-couch-init-1 continuously exits

### DISCUSSION THREAD

**Discussion by ndlang** [Budibase/budibase#3027] [Help] [answered]
I am hosting budi locally and it seems to start successfully but budibase-couch-init-1 keeps stopping. I have tried updating budi and docker (version 4.1.1). I have also re initialized Budibase and I am still having the same issue. Not sure where to go from here, any help would be great. 
Error logs:
{"error":"file_exists","reason":"The database could not be created, the file already exists."}

{"error":"file_exists","reason":"The database could not be created, the file already exists."}

{"error":"file_exists","reason":"The database could not be created, the file already exists."}

sh: fg: line 1: No current job

**Comment by dncpax** [comment, marked as answer]
Hi there. But is BB working? I don't have access to a computer rn but there's a container that exits. I think it only works during startup.

**Reply by shogunpurple** [reply]
This is correct. That container is just an initialisation script that exits when it's done.

**Comment by ndlang** [comment]
Hey,
Thanks for replying. Application starts and I can get to the login page but
when I try to access users or anything to do with users it redirects me
back to the login page and I have to log in again.

**Comment by ndlang** [comment]
I left BB up and running an just logged back in and it seems to be functioning properly now. I do have question though.  I should be able to stop and restart the docker app correct or do I need to leave it up and running all the time?

**Reply by shogunpurple** [reply]
@ndlang you can restart the containers whenever you need to. The docker volumes store data on disk, so the data will be persisted between container starts and stops.

**Comment by ndlang** [comment]
Awesome, thanks!
